core/src/main/java/io/grpc/internal/AbstractStream.java

```diff
@@ -162,30 +162,28 @@ public void messageRead(InputStream is) {
      * Closes this deframer and frees any resources. After this method is called, additional calls
```

Comment:
This is done in the third commit. If this is all eventually to be merged as three separate commits, I can redo the history later to get the comment in the second commit instead.